pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -1607,6 +1611,10 @@ def test_dt64_mul_div_numeric_invalid(self, one, dt64_series):
     @pytest.mark.parametrize('op', ['__add__', '__radd__',
                                     '__sub__', '__rsub__'])
     @pytest.mark.parametrize('tz', [None, 'Asia/Tokyo'])
+    # TODO: What do we want here? We've deprecated adding integers to
```

Comment:
that's why this test isn't parametrized over boxes, is just for the Index class.  (though for many tests in this directory, the reason they are not parametrized over boxes is because I haven't gotten around to it yet)